project-base/app/src/DataFixtures/Demo/ParameterDataFixture.php

```diff
@@ -44,26 +119,213 @@ public function __construct(
+                    $this->getReference(CategoryDataFixture::CATEGORY_ELECTRONICS),
+                    $this->getReference(CategoryDataFixture::CATEGORY_TV),
+                ],
+                akeneoType: Parameter::AKENEO_ATTRIBUTES_TYPE_MULTI_SELECT,
+            ),
+            self::PARAM_WARRANTY_IN_YEARS => new ParameterDataFixtureData(
+                t('Warranty (in years)', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $locale),
+                Parameter::PARAMETER_TYPE_SLIDER,
+                [
+                    $this->getReference(CategoryDataFixture::CATEGORY_PC),
+                ],
+            ),
+            self::PARAM_FOLD => t('Fold', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $locale),
+            self::PARAM_ACTIVE => t('Active', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $locale),
+        ];
+
+        self::$parameterNameCacheByLocale[$locale] = $data;
+
+        return self::$parameterNameCacheByLocale[$locale];
+    }
+
     /**
      * @param \Doctrine\Persistence\ObjectManager $manager
      */
-    public function load(ObjectManager $manager)
+    public function load(ObjectManager $manager): void
     {
+        $firstDomainLocale = $this->domain->getDomainConfigById(Domain::FIRST_DOMAIN_ID)->getLocale();
+        $parameters = $this->getParameterData($firstDomainLocale);
+
+        foreach ($parameters as $parameterDataKey => $parameterDataValue) {
+            $parameterNamesByLocale = [];
+
+            foreach ($this->domain->getAllLocales() as $locale) {
+                $parameterNamesByLocale[$locale] = $this->getParameterNameByReferenceName($parameterDataKey, $locale);
+            }
+
+            if ($parameterDataValue instanceof ParameterDataFixtureData) {
+                $parameter = $this->createParameter(
+                    $parameterDataKey,
+                    $parameterNamesByLocale,
+                    $parameterDataValue->asFilterInCategories,
+                    $parameterDataValue->parameterType,
+                    $parameterDataValue->akeneoType,
+                    $parameterDataValue->parameterGroup,
+                    $parameterDataValue->unit,
+                );
+            } else {
+                $parameter = $this->createParameter(
+                    $parameterDataKey,
+                    $parameterNamesByLocale,
+                );
+            }
+
+            $this->addReference($parameterDataKey, $parameter);
+        }
+
         $parameterColorNamesByLocale = [];
         $parameterMaterialNamesByLocale = [];
 
         foreach ($this->domain->getAllLocales() as $locale) {
             $parameterColorNamesByLocale[$locale] = t('Color', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $locale);
             $parameterMaterialNamesByLocale[$locale] = t('Material', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $locale);
         }
-        $parameterColor = $this->createParameter(
+
+        $this->createParameter(
```

Comment:
these two parameters (color, material) should be already created from the previous foreach, or?

Reply:
yes, you are right, now is everything moved to its correct place